tasks/symfony4/structure.yml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
All this can be done on deploy time, without root privilege. All the parents folder with the right permission have been created so `steamengine_project_user` will be able to add additionnal directories.